Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -53,6 +56,29 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
                let jsonData = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]) {
                 outputText = String(data: jsonData, encoding: .utf8)!
             }
+        case .gpx:
+            var gpxText: String = String("<?xml version=\"1.0\" encoding=\"UTF-8\"?>")
```

Comment:
Maybe it is worth adding a special library (like https://github.com/vincentneo/CoreGPX)?

Reply:
We could, but looking at the examples from that library, it seems like it would be the almost the same number of lines of code (excluding the initial header stuff) since we would still need to parse the RouteResponse on our side to get the lat/lon.

Reply:
Sure, I just have some concerns about generating XML content by hand. All in all, it's just a recommendation as extra dependency has its own pros and cons.

Reply:
Ah I see.

Reply:
I wouldn't include a third part here as @jill-cardamon implementation is good enough for this limiting use case,  I think.